src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  transition-duration: $transition-duration;
+
+  &__title {
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $main-accent-color;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    display: block;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: $font-size-small;
+    line-height: $line-height;
+    margin: 0 0 16px;
+    color: $text-secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: $font-size-small;
+    color: $main-accent-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-weight: 400;
+    font-size: $font-size-medium;
+    color: $text-secondary-color;
+    line-height: 18px;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: 168px;
+    height: 40px;
+    border: 1px solid $link-color;
+    border-radius: $border-radius;
+    background-color: $link-color;
+    font-size: 14px;
+    font-weight: 700;
+    color: $button-bg-color;
+    text-align: center;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-duration: $transition-duration;
+
+    &--active:hover {
+      background-color: $button-bg-color;
```

Comment:
The hover effect for the '.card__buy--active' class correctly changes the background color to '#fff' and the text color to '#00acdc', as specified in the task requirements.